job.sh

```diff
@@ -3,7 +3,11 @@
 source /etc/profile
 source /jlab/remoll/bin/remoll.sh
 
+#curl https://github.com/$3/remoll/blob/$4/geometry/$5 -o $5
+
```

Comment:
I think we should use git clone but only if the argument is non-zero, so
```suggestion
if [ -n "$1" ] ; then
  git clone -b ${1/*:/} https://github.com/${1/:*/} geometry
fi
```